local-running/build.gradle

```diff
@@ -43,6 +43,10 @@ dependencies {
 			project(":libs:verifiable-credentials")
 }
 
+run.doFirst {
+	environment 'AWS_XRAY_CONTEXT_MISSING', 'IGNORE_ERROR'
```

Comment:
This gets rid of all the noisy X-Ray logs. I was looking for a way to turn them off more directly in code, but couldn't find anything.

Reply:
I've spent ages before looking at how to turn x-ray off with some sort of switch - it should be easier